backends/graphics/windowed.h

```diff
@@ -115,6 +115,10 @@ class WindowedGraphicsManager : virtual public GraphicsManager {
 	int getWindowWidth() const { return _windowWidth; }
 	int getWindowHeight() const { return _windowHeight; }
 
+	// New methods to lock and unlock the screen
+	virtual void setLockedScreen(bool val) { _lockedScreen = val; }
```

Comment:
Why do you define getter and setter?
You could put the variable protected and it would be accessible to children classes.